src/style.css

```diff
@@ -1 +1,98 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
```

Comment:
As with the previous font-face declaration, consider using a more descriptive font-family name to avoid conflicts.